doc/admin/deployment.md

```diff
@@ -1053,42 +1053,63 @@ the `[Service]` section before reloading systemd and restarting the
 
 ## System Validation
 
-To validate that the DAOS system is properly installed, the `daos_test`
-suite can be executed. Ensure the DAOS Agent is configured before running
-`daos_test`.  If the agent is using a non-default path for the socket, then
-configure `DAOS_AGENT_DRPC_DIR` in the client environment to point to this new
-location.
+To validate the DAOS system is properly installed, the daos pool autotest
+suite can be executed. Ensure the DAOS Server and Agent are configured
+before running DAOS autotest.
 
 DAOS automatically configures a client with a compatible fabric provider,
-network interface, network domain, CaRT timeout, and CaRT context share address,
-that will allow it to connect to the DAOS system.
+network interface, network domain, CaRT timeout, and CaRT context share
+address, that will allow it to connect to the DAOS system.
 
-The client may not override the fabric provider or the CaRT context share
-address.
 
-A client application may override the three remaining settings by configuring
-environment variables in the client's shell prior to launch.
-
-To manually configure the CaRT timeout, set `CRT_TIMEOUT` such as:
-```
-export CRT_TIMEOUT=5
-```
-To manually configure the network interface, set `OFI_INTERFACE` such as:
-```
-export OFI_INTERFACE=lo
-```
-When manually configuring an Infiniband device with a verbs provider, the network
-device domain is required.  To manually configure the domain, set `OFI_DOMAIN` such as:
-```
-export OFI_DOMAIN=hfi1_0
-```
 ### Launch the client application
```

Comment:
Sounds like we don't really need this sub-section any longer given that the text is shorter.